core/src/appleMain/kotlin/CentralManagerDelegate.kt

```diff
@@ -120,7 +120,9 @@ internal class CentralManagerDelegate : NSObject(), CBCentralManagerDelegateProt
         _state.value = central.state
     }
 
-    // todo: func centralManager(CBCentralManager, willRestoreState: [String : Any])
+    override fun centralManager(central: CBCentralManager, willRestoreState: Map<Any?, *>) {
+        // NO OP
```

Comment:
It isn't clear why this can be a "no-op", it should probably explain it a little bit?

```suggestion
        // No-op: Unnecessary because...
```